### DISCUSSION TITLE
Are the discussion APIs added/available on dock.dev environment?

### DISCUSSION THREAD

**Discussion by joffinjoy** [Sunbird-Lern/Community#26] [Query]
We want to leverage discussion forum APIs for a review comment functionality (for questionSets) in Sunbird Inquiry BB. So are the discussion APIs available in dock.dev environment? If not, how do we add/enable it?

cc: @aks30 @vijiurs, @kiranharidas187 @aishwaryashikshalokam, @priyanka-TL, @VISHNUDAS-tunerlabs, @Shakthieshwari, @Prateek-slokam, @MeghnaSL, @githubpiyush @pradeepkumarcm96

**Comment by VenkatKadiveti** [comment]
@joffinjoy You can follow the steps to attach the discussion forum. 
[Link](https://lern.sunbird.org/use/developer-installation/discussion-forum/installation-guide/discussion-forum-integration-with-any-application)
CC: @reshmi-nair

**Reply by joffinjoy** [reply]
@VenkatKadiveti Let me rephrase my question. I am from dev team, so we first need to know if discussion forum APIs are added/enabled in the dock.dev environment. And if not, what we need is for the owners of discussion forum to add it to the said environment. I am presuming that this will be done by someone from infra team associated with Sunbird Lern BB.

According to my understanding, as devs, we ourselves wouldn't be able to do this. Please correct me if I am wrong.

Basically, we don't know who owns the discussion forum APIs and we would like to get it added to dock.dev environment.

**Reply by VenkatKadiveti** [reply]
Hi @joffinjoy 
You can ask the DevOps team to create a nodebb service and do the deployment. 
Once it is deployed in your environment, then you can able to access the discussion forum apis.

For the local setup of nodebb you can follow this doc ([Nodebb local setup](https://lern.sunbird.org/use/developer-installation/discussion-forum/installation-guide/nodebb-setup)).

If you want to use a discussion forum in any environment. 
1. First need to set up nodebb service. That we can do by deploying the nodebb service using the git repo ([nodebb](https://github.com/NodeBB/NodeBB) with branch v1.18.6).  
2. After then whichever environment you deployed, you can access the nodebb by using the link `https://example-host.com/discussions`
3. After that you can access the apis using the URL  `https://example-host.com/discussions/api/` 

Once you are done with the above steps you can follow these steps [link](https://lern.sunbird.org/use/developer-installation/discussion-forum/installation-guide/discussion-forum-integration-with-any-application).

CC: @reshmi-nair
